components/analytics/compact-dashboard.tsx

```
@@ -0,0 +1,781 @@
+        if (message && typeof message === 'string' &&
+            notableErrorPatterns.some(pattern =>
+                message.toLowerCase().includes(pattern.toLowerCase())
+            )) {
+            return "text-amber-600";
+        }
+
+        return "";
+    };
+
+    // For userReported status, handle undefined/missing properties
+    const getReportStatus = (bot: any, field: string, defaultValue: any = null) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)